src/hardware/targets.json

```diff
@@ -351,12 +351,26 @@
                 "layout_file": "Jumper T-Pro.json",
                 "upload_methods": ["uart", "wifi", "etx"]
             },
+            "t-lite": {
+                "product_name": "Jumper AION T-Lite 2.4GHz TX",
+                "lua_name": "AION T-Lite TX",
+                "layout_file": "Jumper T-Lite.json",
+                "upload_methods": ["uart", "wifi", "etx"]
+            },
             "nano": {
                 "product_name": "Jumper AION Nano 2.4GHz TX",
                 "lua_name": "AION 2G4NANO TX",
                 "layout_file": "Jumper AION Nano.json",
                 "upload_methods": ["uart", "wifi"]
             }
+        },
+        "rx_2400": {
+            "nano": {
+                "product_name": "Jumper AION Nano 2.4GHz RX",
+                "lua_name": "AION 2G4NANORX",
+                "layout_file": "Jumper AION Nano RX.json",
```

Comment:
This is reference the BETAFPV Nano json file.
I think we will be renaming the files to be more Generic later in another PR.